src/shared/templates/shared/partial/popover_tour.html

```diff
@@ -56,12 +56,13 @@
         <div class="row gx-0_5 flex-items-center">
             <div class="col">
                 {% if clusive_user.role == 'TE' or clusive_user.role == 'PA' and has_teacher_resource %}
-                <a href="{% url 'res_reader' resource_id='SSETTINGS' %}">Learn more<span class="sr-only"> about settings</span></a>
+                <a href="{% url 'res_reader' resource_id='SETTINGS' %}">Learn more<span class="sr-only"> about settings</span></a>
```

Comment:
@mbrambilla it actually is "SSETTINGS".  I don't know why.  Logging in as "theresateacher', on the Q/A site:

- https://clusive.qa.cast.org/res_reader/SSETTINGS shows the resource.
- https://clusive.qa.cast.org/res_reader/SETTINGS shows "Server Error".

Reply:
Could be a typo, but it is consistent with the [resources.json](https://github.com/cast-org/clusive/blob/development/resources/resources.json#L70-L74)

Reply:
It was one of the issues I ran into last week -- not finding the resource -- and fixed with this commit:
https://github.com/cast-org/clusive/pull/323/commits/e231b52c8ca40d6c80e7212f399e01225d374e93